Formula/sbcl.rb

```diff
@@ -4,6 +4,7 @@ class Sbcl < Formula
   url "https://downloads.sourceforge.net/project/sbcl/sbcl/2.0.11/sbcl-2.0.11-source.tar.bz2"
   sha256 "87d2aa53cef092119a1c8b2f3de48d209375a674c3b60e08596838013bd7971d"
   license all_of: [:public_domain, "MIT", "Xerox", "BSD-3-Clause"]
+  head "git://git.code.sf.net/p/sbcl/sbcl", shallow: false
```

Comment:
```suggestion
  head "https://git.code.sf.net/p/sbcl/sbcl", shallow: false
```

Reply:
Turns out `using: :git` must be set when using https. https://github.com/Homebrew/homebrew-core/pull/72090